components/_common_/Form/FormCheckBox/FormCheckBox.js

```diff
@@ -18,9 +18,9 @@ FormCheckBox.propTypes = {
 };
 
 FormCheckBox.defaultProps = {
-  checkBox: null,
-  label: null,
-  onChange: null,
+  checkBox: {},
```

Comment:
I prefer more explicit names here. `checkBoxStyles`, etc.  i had to dig into the code to see what this was, couldnt tell from the diff.

take that with a grain of salt though, a preference. ive always found the more explicit the better though

Reply:
@thoughtbyte you're 1000% correct here.

I've copy-pasted the forms from the old codebase, and I'm glad we have coverage on them; however, they will be rewritten from the ground up when we encounter our first form.